sdl_android/src/main/java/com/smartdevicelink/proxy/rpc/MetadataTags.java

```diff
@@ -0,0 +1,87 @@
+package com.smartdevicelink.proxy.rpc;
+
+import com.smartdevicelink.proxy.RPCStruct;
+import com.smartdevicelink.proxy.rpc.enums.MetadataType;
+
+import java.util.Hashtable;
+import java.util.List;
+
+public class MetadataTags extends RPCStruct {
+
+	public static final String KEY_MAIN_FIELD_1_TYPE = "mainField1";
+	public static final String KEY_MAIN_FIELD_2_TYPE = "mainField2";
+	public static final String KEY_MAIN_FIELD_3_TYPE = "mainField3";
+	public static final String KEY_MAIN_FIELD_4_TYPE = "mainField4";
+
+	/**
+	 * Constructs a newly allocated MetadataTags object
+	 */
+	public MetadataTags(){}
```

Comment:
After implementing this into an app I can see the need to have helper methods where developers are able to add only a single MetadataType for each line. For example, public void `setMainField1(MetadataType metadata )`  and the proxy turns it into a list object and sets it.